### ISSUE TITLE
Unable to find Web Essentials via VS2013 Update 5RC

### ISSUE THREAD

**Issue by tbertenshaw** [madskristensen/WebEssentials2013#1847]
There doesn't seem to be a version of webessentials which can be installed with the new VS2013 RC5.

https://www.visualstudio.com/en-us/news/vs2013-update5-vs.aspx

**Comment by martinRocks** [comment]
I'm having the same problem.  There is a version for vs2015 rc.

**Comment by tbertenshaw** [comment]
Yeah quite surprised, there are normally updates released quite soon after rc or beta versions of Visual Studio Service packs.
